lib/src/viam_sdk_impl.dart

```diff
@@ -18,9 +24,37 @@ class ViamImpl implements Viam {
   ViamMovementService? movementService;
   ViamSensorService? sensorService;
   DataService? _dataService;
+  late AppClient _appClient;
+  late DataClient _dataClient;
+
+  ViamImpl();
+
+  ViamImpl.withAccessToken(String accessToken) : _clientChannelBase = AuthenticatedChannel("app.viam.com", 443, accessToken, false) {
```

Comment:
```suggestion
  ViamImpl.withAccessToken(String accessToken) : _clientChannelBase = AuthenticatedChannel('app.viam.com', 443, accessToken, false) {
```

prefer single quotes